docker/build-tools/Dockerfile

```diff
@@ -164,6 +164,13 @@ RUN mv /tmp/upx-${UPX_VERSION}-amd64_linux/upx /usr/bin
 RUN upx --lzma /tmp/go/bin/*
 RUN mv /tmp/go/bin/* ${OUTDIR}/usr/bin
 
+# Add gen-release-notes templates to filesystem
+RUN mkdir -p ${OUTDIR}/usr/share/gen-release-notes
+ADD https://raw.githubusercontent.com/istio/tools/master/cmd/gen-release-notes/templates/minorReleaseNotes.md ${OUTDIR}/usr/share/gen-release-notes
```

Comment:
As a future improvement, a sparse checkout may be better. The templates directory can expand with additional templates as docs requires and right now we have to list them all out explicitly. I'm fine doing this in the future though so approving.

Reply:
Cool. I sort of just want to get the basics working - and that requires live images on gcr.io. Once that works, we can refactor as needed.